src/plugins/intel_gpu/tests/unit/transformations/convert_fc_to_compressed_test.cpp

```diff
@@ -232,6 +233,159 @@ TEST_F(TransformationTestsF, ConvertFCToCompressed7) {
     }
 }
 
+class testSubgraph : public ov::op::util::SubGraphOp {
+public:
+    OPENVINO_OP("testSubgraph", "test", ov::op::util::SubGraphOp);
+
+    testSubgraph() = default;
+
+    testSubgraph(const OutputVector& args, const std::shared_ptr<ov::Model>& body);
+
+    testSubgraph(const NodeVector& args, const std::shared_ptr<ov::Model>& body);
+
+    bool visit_attributes(AttributeVisitor& visitor) override;
+
+    void validate_and_infer_types() override;
+
+    std::shared_ptr<Node> clone_with_new_inputs(const OutputVector& inputs) const override;
+
+private:
+    const ov::Model& body() const {
+        return *m_bodies[0];
+    }
+    ov::Model& body() {
+        return *m_bodies[0];
+    }
+    const std::shared_ptr<ov::Model>& body_ptr() const {
+        return m_bodies[0];
+    }
+    std::shared_ptr<ov::Model>& body_ptr() {
+        return m_bodies[0];
+    }
+
+};
+
+testSubgraph::testSubgraph(const ov::OutputVector& args,
+                                               const std::shared_ptr<ov::Model>& body)
```

Comment:
nit: please fix code style
1. `testSubgraph` -> `TestSubgraph`
2. args alignment

Reply:
Fixed, thank you.